### ISSUE TITLE
Horde, Roundcube CSS and Images not loading

### ISSUE THREAD

**Issue by expssh** [engintron/engintron#245]
Hi. Then trying to login  via webmail.* some of the domain the Horde and Roundcube not loadig CSS styles and images also, but when I open like http://example.com:2095 (with port 2095) it is work. If I Disable Engintron both method works fine, after enable Engintron only with port (2095) images and css loaded. Could you please solve this issue? Thanks

**Comment by brinsleyparagao** [comment]
Hello, i have this issue, but its fixed:
Just disable subdomains cpanel like a: cpanel, webmail and whm
> https://github.com/engintron/engintron/issues/207

**Comment by expssh** [comment]
Hi, You mean I need to delete A record from DNS zone for chose subdomains? Thank you for reply

**Comment by brinsleyparagao** [comment]
In my 5 servers i can only work fine if **Disable** and **Delete**

**Comment by fevangelou** [comment]
@expssh Have your options look like this (under Tweak Settings >> Redirection): http://jmp.sh/ywTWAkA

**Comment by brinsleyparagao** [comment]
@expssh if help in my servers i make this:

> http://imgur.com/JVOPbQ6

**Comment by abkrim** [comment]
Disable? And if customer like use cpanel. webmail, an other, there are not any solution?

```
208.74.121.106 proxy - [08/26/2016:19:01:45 -0000] "GET /cPanel_magic_revision_1410379105/frontend/paper_lantern/libraries/requirejs/optimized/require.min.js HTTP/1.1" 401 0 "http://cpanel.e-XXXXXXX.XXX/cpsess0187918868/frontend/paper_lantern/index.html?login=1&post_login=38904042970074" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36" "-" "-" 8080
```

**Comment by brinsleyparagao** [comment]
@abkrim disable subdomains and not all cpanel system users can be access using cpanel.
exemple;
cpanel.domain.com - no access - no ref to access with this url
domain.com/cpanel - OK - all works fine !

**Comment by abkrim** [comment]
That it's not solution. User like and need access trougth proxsies cpanel. webmail. beacuse it's a option for restrictive firewalls .

Your solution it's same that:

"I've a car. But my car has a rear view mirror protruding fifteen centimeters and width of the road is less.

Solution = not use the car !!

I already know that it works normally. What is spoken here, and maybe not want to try to solve the Depths it is to use the proxy subdomains Cpanel.

**Comment by brinsleyparagao** [comment]
OK, sorry.

**Comment by fevangelou** [comment]
You have to weigh the benefits of using Engintron over the convenience of
using the subdomains for email access.

Unfortunately my dev environments and actual production servers do not
utilize these methods of accessing webmail.

Alternatively you could raise the issue with cPanel and if there is
something I can do about it in Engintron I surely will.

F.

**Comment by abkrim** [comment]
I understand. Best regards

**Comment by Hostert** [comment]
Just to post a follow-up, this issue no longer exists. The access via proxy subdomains works fine in v1.7.2
(Nginx version: 1.10.1)

**Comment by fevangelou** [comment]
It's been fixed either way in the newly released v1.7.3 :)
